cli/client.go

```diff
@@ -45,7 +45,9 @@ import (
 )
 
 const (
-	rdkReleaseURL = "https://api.github.com/repos/viamrobotics/rdk/releases/latest"
+	rdkReleaseURL  = "https://api.github.com/repos/viamrobotics/rdk/releases/latest"
+	defaultNumLogs = 100
+	maxNumLogs     = 10000
```

Comment:
Max here is fairly arbitrary, but I'm trying to avoid CLI users overloading our logs MDB cluster.

Reply:
can add a comment saying that max is arbitrary and default is related to current number of logs returned by a single request